doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,21 +1,21 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 TBC](#A.1)
-* [A.2 TBC](#A.2)
-* [A.3 TBC](#A.3)
-* [A.4 TBC](#A.3)
+* [A.1 Guidance for NFVI Vendors](#A.1)
```

Comment:
Depending on the scope of "NFVI" I'm not sure this is relevant here - do you mean guidance for the providers of the Kubernetes cluster(s)? Does this, as written, imply that the Kubernetes vendor and the NFVI/VIM vendor are the same?

I see the Appendix catering for:
- How cloud native are the CNFs (similar to the discussions in the TUG re. gold, silver, bronze CNFs)?
- For each level of cloud nativeness, what are the features/capabilities of Kubernetes that they need/don't need and how does that affect the compliance of a Kubernetes cluster that is provided for each level?
- For operators who are already running VNFs (and PNFs), what are the considerations and potentially options for adding CNFs into that mix - e.g. relationship with a VIM (or not), relationship with underlay SDN controllers (or not), etc. etc.

Reply:
Agree on all of the above. reworded NFVI to make it more generic and the text will explain what was meant. I will continue to add more details for the framework and diagrams to better clarify my thoughts.

Reply:
@tomkivlin regarding:
Please look at latest edits and let me know what you think.